scaleway/provider.go

```diff
@@ -78,8 +99,20 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		}
 	}
 
+	organization := d.Get("organization").(string)
+
+	if apiKey == "" {
```

Comment:
could we document this functionality [on the provider homepage](https://github.com/terraform-providers/terraform-provider-scaleway/blame/master/website/docs/index.html.markdown#L78)

Reply:
done!